interbotix_ros_xseries/interbotix_xs_sdk/scripts/xs_sdk_sim

```diff
@@ -94,10 +94,17 @@ class InterbotixRobotXS(object):
 
         # populate self.gripper_map
         for gpr, items in grippers.items():
-            self.gripper_map[gpr] = {"horn_radius" : items["horn_radius"],
-                                     "arm_length" : items["arm_length"],
-                                     "left_finger" : items["left_finger"],
-                                     "right_finger" : items["right_finger"]}
+            self.gripper_map[gpr] = {
+                'type': items.get('type', 'swing_arm'),
```

Comment:
String style is inconsistent. Use double quotes in your changes where possible.

Reply:
Resolved